### DISCUSSION TITLE
希望加入禁止自动播放推荐视频的功能

### DISCUSSION THREAD

**Discussion by 921j** [the1812/Bilibili-Evolved#1750] [功能建议 / Ideas]
如题，不知哪次改版B站就悄摸摸把自动播放下一集和自动播放推荐视频合一了，搞得人一时要开自动播放一时要关的挺麻烦，建议根据这个帖子加入禁用自动播放推荐视频又不影响自动播放选集、稍后再看的开关，可以少用一个脚本：
[https://www.bilibili.com/read/cv10762558/](url)
再次感谢作者长久无私的奉献，谢谢！

**Comment by Geeyun-JY3** [comment]
这只是个临时解决方案。并不能保证长久有效。

既然都知道有这个方案了还是建议直接新建一个油猴脚本。

**Comment by the1812** [comment]
多用个脚本也不是啥坏事啊, 要是失效了还能马上关掉/换掉

**Reply by Geeyun-JY3** [reply]
同意，要是做进 Bilibili-Evolved 到时 B 站一改这边又要搞 hoxfix。
既然有现成的方案，直接用就是了。

**Reply by 921j** [reply]
好吧，就看这个脚本能苟多久了，像这样影响用户体验的事搞多了B站迟早受反噬！
